core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java

```diff
@@ -242,7 +260,9 @@ void createSecondaryIndex(String fullKeyspace, String table, Set<String> seconda
     for (String index : secondaryIndexNames) {
       String indexName = String.format("%s_%s_%s", table, INDEX_NAME_PREFIX, index);
       SchemaStatement createIndex =
-          SchemaBuilder.createIndex(indexName).onTable(fullKeyspace, table).andColumn(index);
+          SchemaBuilder.createIndex(indexName)
```

Comment:
Don't we need to quote this index name?

Reply:
We don't need to quote the index name I think.